deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1701,9 +1698,8 @@ def validate_checkers_per_event(events, options, mode):
             def check_events_type(events, ev_type, reg_list=['testkey0']):
                 event_types = Counter(filter_events(events, ".[].data.type"))
 
-                msg = f'Non expected number of events. {event_types[ev_type]} != {len(reg_list)}'
-
-                assert (event_types[ev_type] == len(reg_list)), msg
+                assert (event_types[ev_type] == len(reg_list)
+                        ), f'Non expected number of events. {event_types[ev_type]} != {len(reg_list)}'
```

Comment:
```suggestion
                        ), f"Non expected number of events. {event_types[ev_type]} != {len(reg_list)}"
```
When there is a variable inside a string, we usually use double quotes.

Reply:
Done 25b6b8e9a294aec722b3c8b4f042b11f05f9cc56